test_PragmaTableInfo fails with rqlite 7.2.0

This test failure appeared after upgrade from rqlite 7.1.0 to 7.2.0:
```
======================================================================
FAIL: test_PragmaTableInfo (__main__.SqliteTypeTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "src/test/test_types.py", line 124, in test_PragmaTableInfo
    self.assertEqual(rows,
AssertionError: Lists differ: [(0, [34 chars]s', 'varchar', 0, None, 0), (2, 'f', 'number',[38 chars], 0)] != [(0, [34 chars]s', 'VARCHAR', 0, None, 0), (2, 'f', 'NUMBER',[38 chars], 0)]

First differing element 1:
(1, 's', 'varchar', 0, None, 0)
(1, 's', 'VARCHAR', 0, None, 0)

  [(0, 'i', 'INTEGER', 0, None, 0),
-  (1, 's', 'varchar', 0, None, 0),
?            ^^^^^^^

+  (1, 's', 'VARCHAR', 0, None, 0),
?            ^^^^^^^

-  (2, 'f', 'number', 0, None, 0),
?            ^^^^^^

+  (2, 'f', 'NUMBER', 0, None, 0),
?            ^^^^^^

   (3, 'b', 'BLOB', 0, None, 0)]
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -9,9 +9,9 @@ jobs:
       - checkout
       - run: pip install pytest pytest-cov
       - run: |
-              curl -L https://github.com/rqlite/rqlite/releases/download/v6.7.0/rqlite-v6.7.0-linux-amd64.tar.gz -o rqlite-v6.7.0-linux-amd64.tar.gz
-              tar xvfz rqlite-v6.7.0-linux-amd64.tar.gz
-              cp rqlite-v6.7.0-linux-amd64/rqlited /home/circleci/project/rqlited
+              curl -L https://github.com/rqlite/rqlite/releases/download/v7.2.0/rqlite-v7.2.0-linux-amd64.tar.gz -o rqlite-v7.2.0-linux-amd64.tar.gz
+              tar xvfz rqlite-v7.2.0-linux-amd64.tar.gz
+              cp rqlite-v7.2.0-linux-amd64/rqlited /home/circleci/project/rqlited
       - run:
           name: Run tests
           environment:

--- a/src/test/test_types.py
+++ b/src/test/test_types.py
@@ -40,7 +40,7 @@ class SqliteTypeTests(unittest.TestCase):
             cls.cur.execute("drop table test")
 
     def setUp(self):
-        self.cur.execute("create table test(i integer, s varchar, f number, b blob)")
+        self.cur.execute("create table test(i INTEGER, s VARCHAR, f NUMBER, b BLOB)")
 
     def tearDown(self):
         self.cur.execute("drop table test")
@@ -123,10 +123,10 @@ class SqliteTypeTests(unittest.TestCase):
         rows = self.cur.fetchall()
         self.assertEqual(rows,
             [
-                (0, 'i', 'integer', 0, None, 0),
-                (1, 's', 'varchar', 0, None, 0),
-                (2, 'f', 'number', 0, None, 0),
-                (3, 'b', 'blob', 0, None, 0),
+                (0, 'i', 'INTEGER', 0, None, 0),
+                (1, 's', 'VARCHAR', 0, None, 0),
+                (2, 'f', 'NUMBER', 0, None, 0),
+                (3, 'b', 'BLOB', 0, None, 0),
             ]
         )
 

